### ISSUE TITLE
Players see watchcat NPCs in tab

### ISSUE THREAD

**Issue by RealmportalMC** [CraftG/WatchCat#65]
paper spigot 1.12.2 latest watchcat as of this post

**Comment by NaNaShiSan1** [comment]
It's a feature.

**Comment by RealmportalMC** [comment]
can there be a way to disable them form showing in [tab] tho

**Comment by CraftG** [comment]
yes. You can set KillauraEntity.visible_in_tab_list to false. But it will be bypassable.

**Comment by RealmportalMC** [comment]
why does that make it bypassible? why not just hide from tab?

**Comment by CraftG** [comment]
Because cheat clients can check the tab list.

**Comment by RealmportalMC** [comment]
I dont understand how that stops the anticheat.

**Comment by CraftG** [comment]
If NPC does not appear in TAB list. Cheat client will not attack the NPC, so it can be bypassed.
If you turn it off, Only KillauraEntity can be bypassed.

**Comment by NaNaShiSan1** [comment]
AntiCheat try to detect KillAura by using the bot.
But, hack client has the module called 'AntiBot'. AntiBot blocks from attacking bot.

**Comment by CraftG** [comment]
I only found one AntiBot that can bypass my bot.

**Comment by RealmportalMC** [comment]
Ok i understand now.

**Comment by RealmportalMC** [comment]
1 thing tho: How do i make it so it doesnt show in tab unless they are trying to use killaura on a player?
people said they seen these NPC's as soon as they logged on

**Comment by CraftG** [comment]
not yet. I will add this feature in the next version

**Comment by RealmportalMC** [comment]
ok. i was asking because quite a few people were confused. players didnt know what was going on
